.grabl/automation.yml

```diff
@@ -80,6 +80,7 @@ build:
         export DEPLOY_PIP_USERNAME=$REPO_VATICLE_USERNAME
```

Comment:
`deploy-pip-snapshot` was incorrectly being run concurrently with `build` in CI. So we've now marked build as a dependency of `deploy-pip-snapshot`.